src/lib.rs

```diff
@@ -150,10 +190,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(feature = "nightly", feature(panic_handler))]
 
+#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
```

Comment:
I believe that `no_std` should be usable even during `--test`, so the `not(test)` clause shouldn't be necessary here